pkg/index.yaml

```diff
@@ -119,43 +119,43 @@ entries:
     - https://github.com/confluentinc/cp-docker-images
     - https://github.com/apache/kafka
     urls:
-    - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/kafka-0.5.3.tgz
+    - https://raw.githubusercontent.com/cisco-sso/k8s-charts/master/pkg/kafka-0.5.3.tgz
     version: 0.5.3
   kafka-manager:
   - apiVersion: v1
     appVersion: 1.3.3.16
-    created: 2018-05-18T18:04:59.895471961-05:00
+    created: 2018-05-28T22:04:15.980218585+05:30
     description: A Helm chart for Kafka Manager
     digest: a6e0512342749da84b7798ebba4ed0fac72ad87cab865e3f98958b5649309907
     name: kafka-manager
     urls:
-    - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/kafka-manager-0.1.0.tgz
+    - https://raw.githubusercontent.com/cisco-sso/k8s-charts/master/pkg/kafka-manager-0.1.0.tgz
     version: 0.1.0
   kafka-rest:
   - apiVersion: v1
     appVersion: 4.0.0
-    created: 2018-05-18T18:04:59.895896492-05:00
+    created: 2018-05-28T22:04:15.980607492+05:30
     description: A Helm chart for Kafka REST
     digest: ff324d94a88ecf086574974c3f81bea8b6470b71d9c2995f6d36f1d20f5fa552
     name: kafka-rest
     urls:
-    - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/kafka-rest-0.2.0.tgz
+    - https://raw.githubusercontent.com/cisco-sso/k8s-charts/master/pkg/kafka-rest-0.2.0.tgz
     version: 0.2.0
   kafka-schema-registry:
   - apiVersion: v1
     appVersion: 4.0.0
-    created: 2018-05-18T18:04:59.896432194-05:00
+    created: 2018-05-28T22:04:15.980986622+05:30
     description: A Helm chart for Kafka Schema Registry
     digest: 2429e21310897d891d9df72a60224c69924dad9f149900a44933e2009843cd96
     name: kafka-schema-registry
     urls:
-    - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/kafka-schema-registry-0.2.0.tgz
+    - https://raw.githubusercontent.com/cisco-sso/k8s-charts/master/pkg/kafka-schema-registry-0.2.0.tgz
```

Comment:
^ This will probably break the repo. Please align with the old index on URL stuff.